lib/App/DuckPAN.pm

```diff
@@ -300,12 +300,12 @@ sub execute {
 				elsif($m eq 'ddg' && $ddg){ next }
 				push @modules, $_;
 			}
-			elsif (m/^duckpan|update|(upgrade|reinstall|latest)$/i) {
+			elsif (m/^duckpan|update|(upgrade|(reinstall|latest))$/i) {
```

Comment:
I'm debating whether we really want to support both update (only DuckPAN) and upgrade (DuckPAN + DDG) any longer.  Wouldn't it make sense to do both with update and individually if specified on the command-line?